Makefile.Common

```diff
@@ -150,9 +151,9 @@ misspell:
 misspell-correction:
 	$(MISSPELL_CORRECTION) $(ALL_SRC_AND_DOC)
 
-.PHONY: impi
-impi:
-	@$(IMPI) --local github.com/open-telemetry/opentelemetry-collector-contrib --scheme stdThirdPartyLocal ./...
+.PHONY: gci
+gci:
+	@$(GCI) write ./ -s standard -s default -s "prefix(github.com/open-telemetry/opentelemetry-collector-contrib)"
```

Comment:
Do we still need this after switching to golangci-lint?

Reply:
At least we need to move this flags in the config.

Reply:
There are things that, past midnight, look insurmountable :) I'll muddle through.